packages/client/src/service/txpool.ts

```diff
@@ -379,6 +398,18 @@ export class TxPool {
     }
   }
 
+  pruneBlobsAndProofsCache() {
+    const pruneLength = this.blobsAndProofsByHash.size - BLOBS_AND_PROOFS_CACHE_LENGTH
+    let pruned = 0
+    for (const versionedHash of this.blobsAndProofsByHash.keys()) {
```

Comment:
`Map.keys()` returns in insertion order right? Can this get a comment that this deletes on insertion-order? This is not directly clear, it looks like it "randomly" deletes the order, but since `keys()` is sorted by insertion order this is not the case (it actually deletes based on first-seen).

Reply:
yes it deletes based on first seen